BLE_BatteryLevel/source/main.cpp

```diff
@@ -43,7 +43,7 @@ class BatteryDemo : ble::Gap::EventHandler {
         _ble.init(this, &BatteryDemo::on_init_complete);
 
         _event_queue.call_every(500, this, &BatteryDemo::blink);
-        _event_queue.call_every(1000, this, &BatteryDemo::update_sensor_value);
+        _connection_id = _event_queue.call_every(1000, this, &BatteryDemo::update_sensor_value);
```

Comment:
Please use a different name, this is not a connection id. This is the handle of the reoccurring event that updates the sensor value. What do you think of `_update_sensor_event`

Reply:
I think we should remove this line.

This PR's new logic is, in `onConnectionComplete()` we start the periodic update, and in `onDisconnectionComplete()` we stop the update. (Prior to this PR, we had this line because `update_sensor_value()` ran all the time.)

Reply:
But I agree, the variable name could be `_update_sensor_event` as it's the event ID in the queue

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-ble/compare/3f15044595ebaf5a90644c562cc2904d2087c8f6..d1d8a73fcff47913ce0523927d6cba685d01018e). The line was removed and the variable was renamed as suggested.